### ISSUE TITLE
False positive assigning bool fields in object initialization

### ISSUE THREAD

**Issue by mistiara** [fsprojects/FSharpLint#304]
### Description

Assigning bool *fields* (not properties or named parameters) in object initialization results in false positives for `` `x = true` might be able to be refactored into `x` `` and `` `x = false` might be able to be refactored into `not x` ``.

I couldn't find a duplicate of this issue.

### Minimal repro

![image](https://user-images.githubusercontent.com/1218253/52085631-42ad5c80-25a5-11e9-907f-5f2682861584.png)

**Comment by duckmatt** [comment]
Thank you for raising this! I've [merged in a fix](https://github.com/fsprojects/FSharpLint/pull/321/files) this will go out in version `0.10.7`

**Comment by Thorium** [comment]
I still get this error when assigning boolean to named parameter.

**Comment by knocte** [comment]
@SeanTAllen rule?

**Comment by SeanTAllen** [comment]
@knocte Hints.

I can remove the warning with:

```
      // fsharplint:disable Hints
      let csv =
        CsvFile.Load(Environment.CurrentDirectory + "/" + opts.Datafile, hasHeaders = false)
      // fsharplint:enable
```
